app/views/ui-components/v1/cards/_summary.html.erb

```diff
@@ -127,19 +126,18 @@
 
                   <span class="ttu dg fourteen"><%= @plan.product_type ? @plan.product_type.upcase : "" %></span>
                 </td>
-
                 <td>
-                  <span class="ttu lg twelve">Metal Level</span>
+                  <span class="ttu lg twelve" style="margin-left: -10px">Metal Level</span>
```

Comment:
we should use a new class or premade bootstrap class to do these stylings

Reply:
Removed. Not needed. Another PR previously fixed this issue.